RecoTracker/MeasurementDet/plugins/MeasurementTrackerESProducer.cc

```diff
@@ -47,12 +48,11 @@ MeasurementTrackerESProducer::~MeasurementTrackerESProducer() {}
 std::shared_ptr<MeasurementTracker> 
 MeasurementTrackerESProducer::produce(const CkfComponentsRecord& iRecord)
 { 
-  std::string pixelCPEName = pset_.getParameter<std::string>("PixelCPE");
-  std::string stripCPEName = pset_.getParameter<std::string>("StripCPE");
-  std::string matcherName  = pset_.getParameter<std::string>("HitMatcher");
+  pixelCPEName = pset_.getParameter<std::string>("PixelCPE");
+  stripCPEName = pset_.getParameter<std::string>("StripCPE");
+  matcherName  = pset_.getParameter<std::string>("HitMatcher");
```

Comment:
@ebrondol The `getParameter()` calls are still within `produce()`, even if the variables are members of `MeasurementTrackerESProducer`. I believe @slava77 would like to move the `getParameter()` calls to the constructor (after which `pset_` member can be removed).